### ISSUE TITLE
Upgrade SAI to v1.0.1

### ISSUE THREAD

**Issue by chienbernie** [Azure/sonic-buildimage#631]
I tried to upgrade SAI to v1.0.1, because ASIC is only supporting new SAI version.
I updated /src/SAI to branch v1.0 and v1.0.1 tag and /src/sonic-sairedis to branch v1.0.1
But I got a compile error as below in libsaithrift-dev_1.0.1_amd64.deb.log.
I have checked there is a saithrift folder with files under /src/SAI/test.
What do I need to check? 

make[1]: Leaving directory '/sonic/src/SAI'
 debian/rules build
make[1]: Entering directory '/sonic/src/SAI'
make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
dh build
   dh_testdir
   dh_auto_configure
   debian/rules override_dh_auto_build
make[2]: Entering directory '/sonic/src/SAI'
/usr/bin/make DESTDIR=/sonic/src/SAI/debian saithrift-build 
make[3]: Entering directory '/sonic/src/SAI'
make[3]: *** No rule to make target 'saithrift-build'.  Stop.
make[3]: Leaving directory '/sonic/src/SAI'
debian/rules:26: recipe for target 'override_dh_auto_build' failed
make[2]: *** [override_dh_auto_build] Error 2
make[2]: Leaving directory '/sonic/src/SAI'
debian/rules:23: recipe for target 'build' failed
make[1]: *** [build] Error 2
make[1]: Leaving directory '/sonic/src/SAI'
dpkg-buildpackage: error: debian/rules build gave error exit status 2

**Comment by chienbernie** [comment]
Is there a porting guide for new platform's SAI?

**Comment by lguohan** [comment]
check this branch. https://github.com/qiluo-msft/sonic-buildimage/tree/qiluo/v1.0.1

**Comment by chienbernie** [comment]
Hi lguohan,

I saw there is new v1.0.1 branch of SONiC yesterday and Mellanox has updated new SAI 1.0.1 package.
I downloaded and build this branch today. But sonic-sairedis and SAI are not updated in .gitmodules. And your link is the same. 

Which submodules should be updated for SAI v1.0.1?
Thanks.

[submodule "sonic-sairedis"]
	path = src/sonic-sairedis
	url = https://github.com/Azure/sonic-sairedis
 [submodule "src/SAI"]
	path = src/SAI
	url = https://github.com/opencomputeproject/SAI
	branch = v0.9.4
